client/lib/signup/step-actions/index.js

```diff
@@ -132,9 +132,8 @@ function getNewSiteParams( {
 	const siteTypeTheme = getSiteTypePropertyValue( 'slug', siteType, 'theme' );
 	const selectedDesign = get( signupDependencies, 'selectedDesign', false );
 
-	const shouldSkipDomainStep = ! siteUrl && isDomainStepSkippable( flowToCheck );
```

Comment:
`isDomainStepSkippable` is true only if the flow name is `test-fse` but this flow is already removed. So the value is always false now and we don't need this variable.